### ISSUE TITLE
Abnormal fetal head circumference

### ISSUE THREAD

**Issue by michellejoywang** [obophenotype/human-phenotype-ontology#11336]
**Preferred term label:**
Abnormal fetal head circumference

**Synonyms**
Abnormal fetal HC

**Definition (free text, please give PubMed ID)**
Fetal head circumference is measured in the axial plane at the level of the thalami and cavum septic pellucidi, using the outer border of the bony calvarium. Abnormal fetal head circumference is defined as less than 3%ile or 2 standard deviations below the mean or greater than the 97%ile or 2 standard deviations above the mean. 

**Parent term (use hpo.jax.org/app)**
Abnormality of skull size 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-5177-8109

**Comment by michellejoywang** [comment]
@VarenyaJ PMID!

**Comment by pnrobinson** [comment]
Added as
Any deviation from the normal range of the fetal head circumference, which is typically measured sonographically in the axial plane at the level of the thalami and cavum septic pellucidi, using the outer border of the bony calvarium. Abnormal fetal head circumference is defined as less than the thrid percentile or 2 standard deviations below the mean or greater than the 97the percentile or 2 standard deviations above the mean.
PMID: 35928073
Also added Increased/Decreased terms
